scipy/stats/stats.py

```diff
@@ -101,12 +101,10 @@
 
    pearsonr
    fisher_exact
-   barnard_exact
```

Comment:
This docstring is irrelevant, no? Should we just delete it all?

Reply:
Indeed, it's irrelevant, +1 for deleting.